middleware.ts

```diff
@@ -20,7 +20,7 @@ interface JWTAccessToken extends JwtPayload {
 }
 
 // TODO: These routes will need to be updated.
-const excludedPaths = ['/email', '/favicon.ico', '/_next', '/api'];
+const excludedPaths = ['/email', '/favicon.ico', '/_next', '/api', '/login', '/signup'];
```

Comment:
I added "/login" and "/signup" page to the exclusion list and removed code to return response early when path is excluded, since a check is being done for tokens below. This was causing a 404 error, due to the addition of the "[locale]"